http/api/logging.go

```diff
@@ -1,8 +1,6 @@
 // Copyright (c) Mainflux
 // SPDX-License-Identifier: Apache-2.0
 
-// +build !test
-
```

Comment:
Please don't remove test tags.

Reply:
Ok but can you explain me why this tags are not everywhere? For what this is needed?

Reply:
Replaced by the new one `// go:build`

Reply:
> Ok but can you explain me why this tags are not everywhere? For what this is needed?

It is used to ignore logging and metrics for testing (coverage included) - it is used here: https://github.com/mainflux/mainflux/blob/6483969927adb71736b8d4ed7553f8c2988973ce/scripts/ci.sh#L98